project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=0.13.16
+sbt.version=1.0.3
```

Comment:
g8 plugin does not support sbt 1.0 foundweekends/giter8#318
please enable travis-ci https://github.com/scala/hello-world.g8/issues/3